pythonwhois/net.py

```diff
@@ -91,4 +91,10 @@ def whois_request(domain, server, port=43):
 		if len(data) == 0:
 			break
 		buff += data
-	return buff.decode("utf-8")
+	encodings = ("utf-8", "iso-8859-1")
```

Comment:
Maybe should add WIN-1252 (aka LATIN-1 aka `'cp1252'` in Python), very similar to iso-8859-1 with some differences.